requirements.in

```diff
@@ -4,3 +4,4 @@ strict-rfc3339
 rfc3987
 rfc6266
 requests[socks]
+attrs>=20.3.0
```

Comment:
I'm not clear why this is necessary.

Reply:
@Bjwebb hopefully this is clear in the git commit message